tasks/managed/update-package-collection/update-package-collection.yaml

```
@@ -452,8 +497,13 @@ spec:
           # Extract the NVR (Name-Version-Release) from the SRPM file
           package_nvr=$(basename "$srpm_file" .src.rpm)
 
-          # Display the package NVR
-          echo "Package NVR: $package_nvr"
+          # Add the postfix ",draft_${build_id}" if it is a import draft build
+          if [[ "$KOJI_IMPORT_DRAFT" != "false" && "$PUSH_TYPE" != "promote" ]]; then
```

```suggestion
          if [[ "$KOJI_IMPORT_DRAFT" == "true" ]]; then
```

Maybe we don't need the promote. Just check the if [[ "$KOJI_IMPORT_DRAFT" != "true" ]]; as what you deleted in this PR upon?

---

@xiangge  as I can find below in the push-rpm-koji task:

          if [[ "$KOJI_IMPORT_DRAFT" == "false" ]]; then
              if koji-cmd buildinfo "$PACKAGE_NVR" >/dev/null 2>&1;  then
                  printf "Skip import %s into BREW as it's exist ...\n" "$PACKAGE_NVR"
                  cd .. && rm -rf temp
                  continue
              fi
              draft="false"
              draft_flag=""
          else
              draft="true"
              draft_flag="--draft"
          fi

And there is no field '.pushOptions.koji_import_draft' in the any RPA, seems in most case, the job is using $KOJI_IMPORT_DRAFT" == "true"   and  let draft='true'

---

OK, the MR LGTM then